libpod/info.go

```diff
@@ -21,7 +21,7 @@ type InfoData struct {
 
 // top-level "host" info
 func (r *Runtime) hostInfo() (map[string]interface{}, error) {
-	// lets say OS, arch, number of cpus, amount of memory, maybe os distribution/version, hostname, kernel version, uptime
+	// lets say OS, arch, number of cpus, amount of memory, maybe os distribution/version, hostname, kernel Version, uptime
```

Comment:
Why is kernel Version caps now?

Reply:
that is a result of a refactor for version.  ill revert it.

Reply:
fixed